guide/en/start/doctrine.md

```diff
@@ -0,0 +1,85 @@
+# Doctrine
+
+## Add packages to composer.json
+
+- "doctrine/orm": "^2.7".
+- "symfony/orm-pack": "^1.0".
```

Comment:
What do we need it for? The `orm-pack` seems to add packages that are Symfony specific.

Reply:
We need only the migration packages and doctrine entries.